### ISSUE TITLE
Multiple package.json

### ISSUE THREAD

**Issue by hendrysadrak** [jvitor83/types-autoinstaller#17]
Doesn't work with a project setup with multiple package.jsons. For example monorepos of bigger projects where there is package.json per service.

**Comment by tamj0rd2** [comment]
Is that something npm/yarn is actually able to do without `cd`ing into the other folders first?

**Comment by hendrysadrak** [comment]
Yep. with --prefix flag ([ref](https://stackoverflow.com/questions/14469515/how-to-npm-install-to-a-specified-directory))

**Comment by tamj0rd2** [comment]
Cool, didn't know that! I'll try to do a PR for it in the next few days.

**Comment by tamj0rd2** [comment]
Might take me a while

**Comment by aleclarson** [comment]
I've taken the time to rewrite this extension: https://github.com/aleclarson/vs-auto-typings

It watches all `package.json` modules in the workspace (even deeply nested), except for those inside `node_modules` directories.

It also has tons of other improvements. Check it out, let me know if you hit bugs, and rate it. 😄
